### PR TITLE
Static analysis, simplify boolean conditional expressions

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: src/main/java/games/strategy/triplea/attachments/UnitAttachment.java [hunk 1]
```diff
@@ -3187,8 +3187,8 @@ public String toStringShortAndOnlyImportantDifferences(final PlayerID player, fi
       if ((getBombingMaxDieSides() != -1 || bombingBonus != 0)
           && Properties.getUseBombingMaxDiceSidesAndBonus(getData())) {
         stats.append(bombingBonus != 0 ? bombingBonus + 1 : 1).append("-")
-            .append(getBombingMaxDieSides() != -1 ? getBombingMaxDieSides() + (bombingBonus != 0 ? bombingBonus : 0)
-                : getData().getDiceSides() + (bombingBonus != 0 ? bombingBonus : 0))
+            .append(getBombingMaxDieSides() != -1 ? getBombingMaxDieSides() + (bombingBonus)
+                : getData().getDiceSides() + (bombingBonus))
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/attachments/UnitAttachment.java]: The parentheses around bombingBonus can be removed

File: src/main/java/games/strategy/triplea/attachments/UnitAttachment.java [hunk 2]
```diff
@@ -3284,8 +3284,8 @@ public String toStringShortAndOnlyImportantDifferences(final PlayerID player, fi
...
```
Comment (RoiEXLab) [inline src/main/java/games/strategy/triplea/attachments/UnitAttachment.java]: Parentheses can be removed

**Comment by DanVanAtta** [comment]
Travis is having issues, I don't know if this is worth revisiting. Will follow up in a new branch.

**Comment by DanVanAtta** [comment]
@RoiEXLab  https://github.com/triplea-game/triplea/pull/2787 would remove the now unnecessary parens.

**Pull request merged**